csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -309,92 +316,139 @@ private void TestMultiThreads()
             session.Dispose();
         }
 
-        [x64Fact]
-        private void TestPreTrainedModelsOpset7And8()
-        {
-            var skipModels = new List<String>() {
-                "mxnet_arcface",  // Model not supported by CPU execution provider
-                "tf_inception_v2",  // TODO: Debug failing model, skipping for now
-                "fp16_inception_v1",  // 16-bit float not supported type in C#.
-                "fp16_shufflenet",  // 16-bit float not supported type in C#.
-                "fp16_tiny_yolov2" };  // 16-bit float not supported type in C#.
+        private static Dictionary<string, string> GetSkippedModels()
+        {
+            var skipModels = new Dictionary<string, string>() {
+                { "mxnet_arcface", "Model not supported by CPU execution provider" } ,
+                { "tf_inception_v2", "TODO: Debug failing model, skipping for now" },
+                { "fp16_inception_v1", "16-bit float not supported type in C#." },
+                { "fp16_shufflenet", "16-bit float not supported type in C#." },
+                { "fp16_tiny_yolov2", "16-bit float not supported type in C#." },
+                { "LSTM_Seq_lens_unpacked", "Sequence contains no elements" },
+                { "BERT_Squad", "Could not find an implementation for the node bert / embeddings / one_hot:OneHot(9)" },
+                { "faster_rcnn", "Length of memory(2611200) must match product of dimensions(3)" },
+                { "mask_rcnn", "Length of memory (3456000) must match product of dimensions (3)" },
+                { "mask_rcnn_keras", "Length of memory (3145728) must match product of dimensions (3)" },
+                { "mlperf_resnet", "Value cannot be null" },
+                { "mlperf_ssd_mobilenet_300", "Could not find file output_0.pb" },
+                { "yolov3", "Length of memory(519168) must match product of dimensions(3)" }
```

Comment:
The `"Length of memory..."` error message is because the dimensions of the tensor don't match the length of the tensor. Some of these models can probably be removed from the skip-list with a minor fix. (E.g  I was able to run **yolov3** successfully here --> https://github.com/microsoft/onnxruntime/issues/1770)